lnbits/core/templates/admin/_tab_funding.html

```diff
@@ -25,117 +25,70 @@ <h6 class="q-my-none">Wallets Management</h6>
           </ul>
           <br />
         </div>
-      </div>
-      <div class="row">
+        {% if LNBITS_NODE_UI_AVAILABLE %}
         <div class="col">
-          <div v-if="'{{LNBITS_NODE_UI_AVAILABLE}}' === 'True'">
-            <div>Node Management</div>
-            <q-toggle
-              label="Node UI"
-              v-model="formData.lnbits_node_ui"
-            ></q-toggle>
-            <q-toggle
-              v-if="formData.lnbits_node_ui"
-              label="Public node UI"
-              v-model="formData.lnbits_public_node_ui"
-            ></q-toggle>
-            <q-toggle
-              v-if="formData.lnbits_node_ui"
-              label="Transactions Tab (Disable on large CLN nodes)"
-              v-model="formData.lnbits_node_ui_transactions"
-            ></q-toggle>
-            <br />
-          </div>
+          <p>Node Management</p>
+          <q-toggle
+            label="Node UI"
+            v-model="formData.lnbits_node_ui"
+          ></q-toggle>
+          <q-toggle
+            v-if="formData.lnbits_node_ui"
+            label="Public node UI"
+            v-model="formData.lnbits_public_node_ui"
+          ></q-toggle>
+          <br />
+          <q-toggle
+            v-if="formData.lnbits_node_ui"
+            label="Transactions Tab (Disable on large CLN nodes)"
+            v-model="formData.lnbits_node_ui_transactions"
+          ></q-toggle>
           <p v-else>Node Management not supported by active funding source</p>
         </div>
+        {% endif %}
```

Comment:
this doesnt behave the same as before. the the `p` should show up when theres no node management available, so you'd need `{% else %}

Reply:
good catch! missed that, thanks!